apps/cli/src/commands/bot.ts

```diff
@@ -30,6 +30,12 @@ export default {
       throw new Error("Please define a SEED env variable to run this bot.");
     }
 
+    if (arg.currency) {
+      // Remapping to match arg format in libs/ledger-live-common/src/bot/cli.ts
+      arg.filter = { currencies: arg.currency };
```

Comment:
i think it should be currencies: [arg.currency] but i'm not sure

Reply:
You're right :+1: thanks.

But it already works without it :open_mouth: which is why I didn't notice.
Looking at it, it's actually a very lucky case, because I'm seeing that only `.length` and `.includes` are used on `currencies`, which work for both string and array :smile:

Reply:
type doesn't cry but on code execution it's another story :smile: